### ISSUE TITLE
Does this module support HTTP2?

### ISSUE THREAD

**Issue by 0079123** [arut/nginx-rtmp-module#1262]
What do I need to do?

**Comment by shadoxx** [comment]
Hi @0079123,

What are you trying to accomplish? This module enables an RTMP/RTP backend for Nginx. It does support HTTP/2 streaming with HLS through the HTTP/2 module inside of Nginx. This project also has the benefit of being extremely well documented: https://github.com/arut/nginx-rtmp-module/wiki/Directives

If you're trying to accomplish HLS streaming over HTTP/2, please look at the `application hls {}` directive in the Wiki: https://github.com/arut/nginx-rtmp-module/wiki/Directives#hls

**Comment by campones** [comment]
I don't think there is anything related to http2 in the hls directive.

**Comment by shadoxx** [comment]
@campones Correct. The HTTP2 support is completely independent of this module. Just adding the proper HTTP2 directives to the server{} block of your Nginx configuration is sufficient enough to enable HTTP2 support for HLS/M3U8 streams.

**Comment by campones** [comment]
I know
